src/test/shared/codelens/localLambdaRunner.test.ts

```diff
@@ -53,6 +56,10 @@ class FakeBasicLogger implements BasicLogger {
 }
 
 describe('localLambdaRunner', async () => {
+
+    const tempDir: string = await fsUtils.makeTemporaryToolkitFolder()
+    ExtensionDisposableFiles.getInstance().addFolder(tempDir)
```

Comment:
Does this clean up the folders? Other tests have explicitly cleaned up `tempDir` in an `after`/`afterEach`. 
Along with that the tempDir is typically assigned in `before`/`beforeEach`

Reply:
:+1: on before. Adding this to ExtensionDisposableFiles should automatically clean up the directory upon process exit.

Reply:
✔️